hapi-proto/src/main/proto/ScheduleGetInfo.proto

```diff
@@ -43,6 +43,7 @@ message ScheduleInfo {
   bytes transactionBody = 4; // The transaction serialized into bytes that must be signed
   KeyList signers = 5; // The keys that have provided signatures so far for the Scheduled TX
   Key adminKey = 6; // The Key which is able to delete the Scheduled Transaction if set
+  bytes memo = 7; // The additional bytes that carry additional information
```

Comment:
the word `additional` seems redundant here. We could use: `The additional informational bytes that were submitted as part of the creation of the entity`